packages/mui-base/src/SliderUnstyled/SliderUnstyled.d.ts

```diff
@@ -93,6 +94,11 @@ export interface SliderUnstyledTypeMap<P = {}, D extends React.ElementType = 'sp
      * @returns {string}
      */
     getAriaValueText?: (value: number, index: number) => string;
+    /**
+     * [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes) applied to the `input` element.
+     * @default {}
+     */
+    inputProps?: InputBaseComponentProps;
```

Comment:
Let's add a new key inside `componentsProps` instead of adding a new prop.

Reply:
Thanks for the review. I hesitated to add it there, but I felt like `componentsProps` should be consistent with `components`. But I preferred to go for the `inputProps` to be consistent with the API of other existing components like `TextField` and also because I'm not adding the corresponding `componentsProps.Input` property.

What do you think ?

Reply:
I would add the option both in the `components` and `componentsProps`. We plan to move all components to support this new API (`components` and `componentsProps`). At least this will be true for the base components. Regarding the key, we have two options: `hiddenInput` or `input`. cc @michaldudak what option do you think it's more intuitive?

Reply:
`input` is fine.